File: eSalon/UI/esalon_desktop/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? ime;
  String? prezime;
  String? korisnickoIme;
  String? email;
  String? telefon;
  bool? jeAktivan;
  DateTime? datumRegistracije;
  List<String>? uloge;
  String? slika;
  

  //Korisnik();
    Korisnik({
    this.korisnikId,
    this.ime,
    this.prezime,
    this.korisnickoIme,
    this.email,
    this.telefon,
    this.jeAktivan,
    this.datumRegistracije,
    this.uloge,
    this.slika,
  });

  factory Korisnik.fromJson(Map<String, dynamic> json) =>
      _$KorisnikFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikToJson(this);

}

File: eSalon/eSalon.Model/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class Korisnik
    {
        public int KorisnikId { get; set; }

        public string Ime { get; set; } = null!;

        public string Prezime { get; set; } = null!;

        public string KorisnickoIme { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string? Telefon { get; set; }

        public bool? JeAktivan { get; set; }

        public DateTime DatumRegistracije { get; set; }

        public List<string> Uloge { get; set; } = new List<string>();

        public byte[]? Slika { get; set; }
      
    }
}
